test/queueTest.ts

```diff
@@ -1,63 +1,51 @@
 import assert from 'node:assert'
 import * as helper from './testHelper.ts'
-import { PgBoss, states } from '../src/index.ts'
+import { states } from '../src/index.ts'
 
 describe('queues', function () {
   it('should create a queue', async function () {
-    this.boss = await helper.start({ ...this.bossConfig, noDefault: true }) as PgBoss
+    this.boss = await helper.start({ ...this.bossConfig, noDefault: true })
 
     await this.boss.createQueue(this.schema)
   })
 
   it('createQueue should work if queue already exists', async function () {
-    this.boss = await helper.start({ ...this.bossConfig, noDefault: true }) as PgBoss
+    this.boss = await helper.start({ ...this.bossConfig, noDefault: true })
 
     await this.boss.createQueue(this.schema)
     await this.boss.createQueue(this.schema)
   })
 
   it('should reject a queue with invalid characters', async function () {
-    this.boss = await helper.start({ ...this.bossConfig, noDefault: true }) as PgBoss
-    const queue = `*${this.bossConfig.schema}`
-
-    assert.rejects(async () => {
-      await this.boss.createQueue(queue)
-    })
-  })
-
-  it('should reject a queue that starts with a number', async function () {
-    this.boss = await helper.start({ ...this.bossConfig, noDefault: true }) as PgBoss
-    const queue = `4${this.bossConfig.schema}`
-
-    assert.rejects(async () => {
+    await assert.rejects(async () => {
+      this.boss = await helper.start({ ...this.bossConfig, noDefault: true })
```

Comment:
You should keep inside the assertion only the thing being tested (in this case the createQueue method).

If for some reason, .start fails, the test would keep passing, even if it's not working the way it's desired.